shared/src/main/kotlin/org/javacs/kt/classpath/BackupClassPathResolver.kt

```diff
@@ -27,24 +28,13 @@ private fun findLocalArtifact(group: String, artifact: String) =
 private fun tryFindingLocalArtifactUsing(@Suppress("UNUSED_PARAMETER") group: String, artifact: String, artifactDirResolution: LocalArtifactDirectoryResolution): Path? {
     val isCorrectArtifact = BiPredicate<Path, BasicFileAttributes> { file, _ ->
         val name = file.fileName.toString()
-        when (artifactDirResolution.buildTool) {
-            "Maven" -> {
-                val version = file.parent.fileName.toString()
-                val expected = "${artifact}-${version}.jar"
-                name == expected
-            }
-            else -> name.startsWith(artifact) && name.endsWith(".jar")
-        }
+        name == "$artifact-${KotlinVersion.CURRENT}.jar"
```

Comment:
This uses the language server's version of Kotlin, right? This might go wrong if the user has a different version of Kotlin installed than what the language server uses.

Reply:
Good point. However, I think we need a better way to figure out the Kotlin version in the project on which the LSP runs, maybe via extracting the version from the Gradle configuration, if possible. 
The current implementation uses the first version it finds, which also doesn't necessarily match the Kotlin version used in the project.